docs/cdn_commands.md

```diff
@@ -102,3 +102,10 @@ cwr('recordEvent', {type: 'your_event_type', data: {field1: 1, field2: 2}})
 ```
 awsRum.recordEvent('your_event_type', {field1: 1, field2: 2})
 ```
+
+## RecordEventOptions
+
+| Field Name | Type | Default | Example | Description |
+| --- | --- | --- | --- | --- |
+| isCandidate | Boolean | FALSE | In this example, RUM will cache all 'running_max' candidates, but only call the final value=15 to PutRumEvents at the end of the Page's lifecycle. This is a very frugal solution if you need to keep a running count, but only want to publish the event at the latest possible time. <br/><br/>aws.recordEvent('final_click_count', {value: 5}, {isCandidate: true, replaceFirstMatch: true})<br/>aws.recordEvent('final_click_count', {value: 10}, {isCandidate: true, replaceFirstMatch: true})<br/>aws.recordEvent('final_click_count', {value: 15}, {isCandidate: true, replaceFirstMatch: true}) | If true, then RUM will not dispatch the candidate event to PutRumEvents until the next visibilitychange event, which only occurs when the page is hidden or closed.<br/><br/>Event candidates are prioritized over regular events in that final batched call because that is the only opportunity for them to be reported. <br/><br/>To provision more cache space for candidates, then increase candidatesCacheSize. |
+| replaceFirstMatch | Boolean | FALSE | In this example, RUM will cache all 'running_click_count' events, but only a last event will be reported in the next batch. If the previous matching event types were already send to PutRumEvents, then additional events will be recorded. This is a good way to minimize the risk of dropping events while still reducing some redundant data. <br/><br/>aws.recordEvent('running_click_count', {value: 5}, {replaceFirstMatch: true})<br/>aws.recordEvent('running_click_count', {value: 10}, {replaceFirstMatch: true})<br/>aws.recordEvent('running_click_count', {value: 15}, {replaceFirstMatch: true}) | If true, then RUM will replace the first matching event type in the event cache. If isCandidate=true as well, then the first matching candidate type is replaced in the candidate cache. |
```

Comment:
suggestion: is there a way you can make this example more explicit?  i think it might be hard to follow. 
for example: 
> but only a last event will be reported in the next batch

Can we use an example alongisde the one you have already to explain this?

Reply:
Removed public docs in rev2